pkg/placement/external-name.go

```diff
@@ -16,31 +16,10 @@ limitations under the License.
 
 package placement
 
-import (
-	"github.com/kcp-dev/logicalcluster/v3"
-)
-
 // ExternalName identifies a cluster-scoped object of some implicit kind
 type ExternalName struct {
-	// Cluster identifies the cluster.  It is the one-part ID, not a path.
-	Cluster logicalcluster.Name
+	// Cluster identifies the cluster.  It is the space ID.
```

Comment:
I would have said "space name" rather than "space ID", since it really is the Name not the UID of a Space object and "space name" is more commonly used for this elsewhere in this module.